Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/InteractableToggleCollection.cs

```diff
@@ -23,7 +23,51 @@ public class InteractableToggleCollection : MonoBehaviour
         public Interactable[] ToggleList
         {
             get => toggleList;
-            set => toggleList = value;
+            set 
+            {
+                if (value == null)
+                {
+                    return;
+                }
+
+                if (toggleList != value)
+                {
+                    if (toggleList != null)
+                    {
+                        // Destroy all listeners on previous toggleList
+                        for (int i = 0; i < toggleActions.Count; ++i)
+                        {
+                            toggleList[i]?.OnClick.RemoveListener(toggleActions[i]);
+                        }
+                        listenersAdded = false;
+                        toggleActions.Clear();
```

Comment:
This is redundant code with OnDestroy. Move it to dedicated function

Reply:
Fixed, added RemoveSelectionListeners()